pkg/recipes/driver/terraform.go

```diff
@@ -249,6 +249,11 @@ func (d *terraformDriver) GetRecipeMetadata(ctx context.Context, opts BaseOption
 	return recipeData, nil
 }
 
+func (d *terraformDriver) FindSecretIDs(ctx context.Context, envConfig recipes.Configuration, definition recipes.EnvironmentDefinition) (string, error) {
+
+	return recipes.GetSecretStoreID(envConfig, definition.TemplatePath)
```

Comment:
Why does it call into another function in recipes pkg? Any reason why we aren't moving the implementation here?

Reply:
currently we cannot, because we are using this function in config.go to get the template path prefix, we can move the implementation here when we have containerization.

Reply:
Ah right. Can you please add that in the comment and call it out in the ?